workflows/SUEP_coffea_ZH_simple.py

```diff
@@ -841,17 +877,19 @@ def doBTagWeights(self, events, jetsPre, wp="L"):
         corrector = correctionlib.CorrectionSet.from_file(btagfile)
         correctorL = correctionlib.CorrectionSet.from_file(btagfileL)
         SF = {}
-        SF["central"] = corrector["deepJet_comb"].evaluate("central", wp, hadronFlavourLightAsB, np.abs(jets.eta),jets.pt) # SF per jet, later argument is dummy as will be overwritten next
-        SF["central"] = np.where(abs(jets.hadronFlavour) == 0, correctorL["deepJet_incl"].evaluate("central", wp, hadronFlavourBCAsLight, np.abs(jets.eta),jets.pt) ,SF["central"])
+        flattened_pt = np.array(jets.pt)
+        flattened_eta = np.array(np.abs(jets.eta))
```

Comment:
Running abs first is probably more efficient in fact, so this is likely for the best